cachelib/navy/common/FdpNvme.cpp

```diff
@@ -49,13 +59,7 @@ int FdpNvme::allocateFdpHandle() {
   return static_cast<int>(phndl);
 }
 
-void FdpNvme::initializeFDP(const std::string& bdevName) {
-  nvmeData_ = readNvmeInfo(bdevName);
-
-  Buffer buffer = nvmeFdpStatus();
-  struct nvme_fdp_ruh_status* ruh_status =
-      reinterpret_cast<struct nvme_fdp_ruh_status*>(buffer.data());
-
+void FdpNvme::initializeFDPHandles(struct nvme_fdp_ruh_status* ruh_status) {
   if (!ruh_status->nruhsd) {
     throw std::invalid_argument("Failed to initialize FDP; nruhsd is 0");
   }
```

Comment:
Line 66 should be resize instead of reserve.

Reply:
Is this review comment related to only this PR or it is applicable to original FDP PR also? 
If it is applicable to FDP PR also, should we take this change seperately?

Reply:
I think this one needs to be changed otherwise the new test will be failing. When running the unit test internally I observed what seems like an index our of range fault and I suspected this was the problem.